packages/fakta/arbeid-og-inntekt/src/components/ArbeidsforholdRad.tsx

```diff
@@ -251,31 +211,24 @@ export const ArbeidsforholdRad = ({
           />
         )}
       </Table.DataCell>
-      <Table.DataCell className={erRadÅpen ? styles.colTopPadding : undefined}>
-        {erRadÅpen && <Label size="small">{arbeidsgiverNavn}</Label>}
-        {!erRadÅpen && <BodyShort size="small">{arbeidsgiverNavn}</BodyShort>}
+      <Table.DataCell>
+        <BodyShort>{arbeidsgiverNavn}</BodyShort>
       </Table.DataCell>
-      <Table.DataCell className={erRadÅpen ? styles.colTopPadding : undefined}>
+      <Table.DataCell>
         <BodyShort>
-          {periode?.fom && (
-            <PeriodLabel
-              dateStringFom={periode.fom}
-              dateStringTom={periode.tom !== TIDENES_ENDE ? periode.tom : undefined}
-            />
-          )}
-          {!periode && '-'}
+          {periode?.fom ? periodFormat(periode.fom, periode.tom !== TIDENES_ENDE ? periode.tom : undefined) : '-'}
         </BodyShort>
       </Table.DataCell>
-      <Table.DataCell className={erRadÅpen ? styles.colTopPadding : undefined}>
+      <Table.DataCell>
         <BodyShort>
           <FormattedMessage id={finnKildekode(erManueltOpprettet, arbeidsforholdForRad.length > 0)} />
         </BodyShort>
       </Table.DataCell>
-      <Table.DataCell className={erRadÅpen ? styles.colTopPadding : undefined}>
+      <Table.DataCell>
         <BodyShort>
-          {arbeidsforholdForRad.length < 2 && inntektsmeldingerForRad.length === 1 && (
-            <DateLabel dateString={inntektsmeldingerForRad[0].motattDato} />
-          )}
+          {arbeidsforholdForRad.length < 2 &&
+            inntektsmeldingerForRad.length === 1 &&
+            dateFormat(inntektsmeldingerForRad[0].motattDato)}
```

Comment:
Dette blir personleg smak, men eg syns det er lettare å lesa når ein returnerar komponent her. Var litt usikker lenger opp her, der ein brukar periodFormat i staden for komponent. Men når eg kom til denne, som har litt lenger if-sjekk så syns eg forskjellen i lesbarheit var større. (Men eg meinar ikkje at du bør endra, ville berre kommentera det :)

Reply:
Grunnen til at jeg byttet helt er fordi PeriodLabel wrapper datoen med BodyShort, mens DateLabel ikke gjør det. Så for å gjøre det likt og basic valgte jeg å bruke funksjonene direkte. Men i utgangspunktet er jeg enig i at komponenten er mest lettleselig :D Jeg tenker vi bør fjerne bodyshort-wrapperen i PeriodLabel. Men da bør vi sjekke om det påvirker k9 også

Reply:
ref. https://github.com/navikt/fp-frontend/pull/6360 så er dette fikset nå